dawn/main/ansible/Ansible.js

```diff
@@ -55,15 +77,40 @@ ipcMain.on('stateUpdate', (event, data) => {
   });
 });
 
+setInterval(() => {
+  const message = new DawnData({
+    student_code_status: 1 - test,
+    gamepads: {
+      index: 1,
+      axes: [0.2],
+      buttons: [true],
+    },
+    peripheral_names: ['Test'],
+  });
+  const buffer = message.encode().toBuffer();
+  client.send(buffer, clientPort, hostname, (err) => {
+    if (err) {
+      console.error('UDP socket error on send:', err);
+    }
+  });
+}, 1000);
+
 /**
  * Handler to receive messages from the robot Runtime
  */
 server.on('message', (msg) => {
   try {
-    const data = RuntimeData.decode(msg).toRaw();
-    console.log(`Dawn received: ${JSON.stringify(data)}\n`);
+    const data = RuntimeData.decode(msg);
+    console.log(`Dawn received Runtime Information at ${(new Date).toISOString()}`);
+    test = 1 - test;
```

Comment:
The scope of `test` here is global (test was defined on global scope), so I think it should be fine.